test/tools/utils.js

```diff
@@ -246,12 +246,25 @@ class EventCollector {
   }
 }
 
+function getSymbolFrom(target, symbolName, assertExists = true) {
```

Comment:
Cool util! 🎉

Reply:
Thanks, and good catch on the other ticket I'll be sure to note it in the merge